tests/test_features_generator.py

```diff
@@ -0,0 +1,117 @@
+from collections import Counter
+
+import pytest
+import numpy as np
+from pandas import DataFrame
+from numpy.testing import assert_array_equal
+
+from sklearn_pandas import DataFrameMapper
+from sklearn_pandas.features_generator import gen_features
+
+
+class MockClass(object):
+
+    def __init__(self, value=1, name='class'):
+        self.value = value
+        self.name = name
+
+
+class MockTransformer(object):
+
+    def __init__(self):
+        self.most_common_ = None
+
+    def fit(self, X, y=None):
+        [(value, _)] = Counter(X).most_common(1)
+        self.most_common_ = value
+        return self
+
+    def transform(self, X, y=None):
+        return np.asarray([self.most_common_] * len(X))
+
+
+@pytest.fixture
+def simple_dataset():
+    return DataFrame({
+        'feat1': [1, 2, 1, 3, 1],
+        'feat2': [1, 2, 2, 2, 3],
+        'feat3': [1, 2, 3, 4, 5],
+    })
+
+
+@pytest.mark.parametrize('columns', [['colA', 'colB', 'colC']])
+def test_generate_features_with_default_parameters(columns):
+    """
+    Tests generating features from classes with default init arguments.
+    """
+    feature_defs = gen_features(columns=columns, classes=[MockClass])
+    assert len(feature_defs) == len(columns)
+
+    feature_dict = dict(feature_defs)
+    assert columns == sorted(feature_dict)
```

Comment:
Can you write `assert sorted(columns) == sorted(feature_dict.keys())`? I didn't know that `sorted(dict)` returns its keys sorted and it's not evident.